package.json

```diff
@@ -45,6 +45,7 @@
     "url": "https://github.com/ckeditor/ckeditor5-build-classic.git"
   },
   "scripts": {
+    "preversion": "npm run build; if [ -n \"$(git status --porcelain)\" ]; then git add -u build && git commit -m 'Internal: Build.'; fi",
```

Comment:
This will fail if the change was in some other directory than `build/`. If you `git add build`, then also check the status of that directory.

Besides, what does `-u` do?

Reply:
> Besides, what does -u do?

Updated files which are tracked by git.

Reply:
Yes, I can use `git add --help`. But that still means nothing to me :P And since you used it I assume that we need to use it so that it does something important.

Reply:
![image](https://user-images.githubusercontent.com/2270764/30367720-498614e4-986f-11e7-870c-2baa7396986d.png)

`-u` flag updates only files tracked by Git. It means if some new file occurs and it does not track by Git, it won't be added to staged chunks.